src/pageTemplates/DetailsPage.tsx

```diff
@@ -0,0 +1,236 @@
+import React from "react";
+// import { PageProps } from "gatsby";
```

Comment:
Shall we remove this file?
If you need code, you can keep this file handy elsewhere?

Reply:
Yes I cleanup the code so we can start adding new code as per use of task.